corona-warn-app/cwa-app-android
# Adapt Vaccination Color + Bring Back EU Stars (EXPOSUREAPP-9076)

Solves a problem which switched the color of the certificate details screen to blue for non final shots once the final shots was also received.

To test: 
1. Scan a new certificate (1 of 2)
2. Scan the final shot (2 of 2) and check the background color of the certificate details screen of the certificate (1 of 2) -> Should stay grey
3. Check that certificates for JJ are displayed blue


Also: Check if the stars are back in the certificate detail view
